src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests.cs

```diff
@@ -13,6 +13,60 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
     [CompilerTrait(CompilerFeature.IOperation)]
     public partial class IOperationTests : SemanticModelTestBase
     {
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.RefLocalsReturns)]
+        [Fact]
+        public void SuppressNullableWarningOperation()
+        {
+            var comp = CreateCompilation(@"
+class C
+{
+#nullable enable
+    void M(string? x)
+    /*<bind>*/{
+        x!.ToString();
```

Comment:
>x [](start = 8, length = 1)

We should have a simple flow graph test for a scenario with control flow inside the underlying expression.